webpack/react_app/components/TargetingHosts/components/HostItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ActionButtons } from 'foremanReact/components/common/ActionButtons/ActionButtons';
+import HostStatus from './HostStatus';
+
+const HostItem = ({ name, link, status, actions }) => {
+  const renderName = () => {
```

Comment:
should it be renamed to `hostLink` ? also it doesn't have to be a function,
can be decleared as 

```js
const hostLink = link ? <a href={link}>{name}</a> : <a href="#" className="disabled">{name}</a>
```